db/model/room.js

```diff
@@ -33,6 +34,14 @@ module.exports = function room(seq, dataTypes) {
       },
       comment: 'Create a named room ',
     },
+    settings: {
+      type: dataTypes.ARRAY(dataTypes.JSONB),
+      allowNull: true,
+      validate: {
+        contains: u.validateSettingsArray,
+      },
+      comment: 'Key/Value pairs for user specific settings',
```

Comment:
@BrandonMurnane I think that raises a good point here. The room type stores the initial values that will only be accessed once when the room is created, but as a bot developer you may hit the room settings often.

So since we are inheriting them anyway we could do the transform from [{key: name, value:info}] to {key:info} for the ease of the bot developer